spec/linter-stylelint-spec.js

```diff
@@ -263,4 +265,27 @@ describe('The stylelint provider for Linter', () => {
       expect(messages.length).toBe(0);
     });
   });
+
+  describe('will autofix what errors it can', () => {
+    it('saves fixed css if attemptFix is true', async () => {
+      atom.config.set('linter-stylelint.attemptFix', true);
+      atom.config.set('linter-stylelint.disableWhenNoConfig', false);
+      const fixedEditor = await atom.workspace.open(doneFixCSS);
```

Comment:
Since this file is pretty small, let's just embed the text in here instead of creating a separate file.